src/js/api/config.js

```diff
@@ -105,6 +109,33 @@ define([
             config.skin = config.skin.replace('.xml', '');
         }
 
+        if (config.playbackRateControls) {
+            var playbackRateOptions;
+
+            var validRatesFilter = (value) => { return _.isNumber(value) && value >= 0.25 && value <= 4; };
+
+            if (_.isBoolean(config.playbackRateControls)) {
+                playbackRateOptions = [0.5, 1, 1.25, 1.5, 2];
```

Comment:
Initialize `playbackRateOptions` to the defaults. That way, you only need to check `_.isArray(config.playbackRateControls)`.